homeassistant/components/dlink/data.py

```diff
@@ -0,0 +1,57 @@
+"""Data for the D-Link Power Plug integration."""
+from __future__ import annotations
+
+from datetime import datetime
+import logging
+import urllib
+
+from pyW215.pyW215 import SmartPlug
+
+from homeassistant.util import dt as dt_util
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class SmartPlugData:
```

Comment:
In the future I'd switch this to use DataUpdateCoordinator to avoid the manual logging